### PR TITLE
FHAC-475: SCA Profile in Maven

### PR DESCRIPTION
**Pull request by christophermay07**
- .gitignore and pom.xml cleanup
- build-tools (SCA support files) cleanup
- Removed unused Jetty plugin
- Created SCA profile, which activates FindBugs+FindSecBugs, PMD, Checkstyle, and OWASP Dependency Check

### CODE REVIEW

**Comment by christophermay07** [comment]
retest this please

**Comment by nareshcgi** [comment]
restest this please

**Comment by nareshcgi** [comment]
retest this please

**Comment by christophermay07** [comment]
retest this please

File: pom.xml [hunk 1]
```diff
@@ -230,23 +372,34 @@ By advancing the adoption of interoperable health IT systems and health informat
             </properties>
             <build>
                 <plugins>
-                    <!-- <plugin> <groupId>com.github.github</groupId> <artifactId>downloads-maven-plugin</artifactId>
```
Comment (nareshcgi) [inline pom.xml:233]: Please remove the commented code.

**Comment by nareshcgi** [comment]
Reviewed the PR and looks good.

**Comment by alameluchidambaram** [comment]
Reviewed PR and looks good.

**Comment by sailajaa** [comment]
Reviewed PR and good with merge

**Pull request merged**